zbus/src/address.rs

```diff
@@ -987,7 +987,7 @@ mod tests {
                 port: 2934
             })
             .to_string(),
-            "vsock:cid=98,port=2934,guid=123",
+            "vsock:cid=98,port=2934", // no support for guid= yet..
```

Comment:
nit: commit log summarizes the benefit, not the change itself. We ask people to [follow these rules](https://wiki.gnome.org/Git/CommitMessages) so we should follow them too.

Reply:
ok

Reply:
You added details about the error being fixed (which is really nice) but did not address my comment. Something like:

```
zb,test: Don't expect guid in vsock address

<what you have right now>
```